src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const populationList = document.querySelectorAll('.population');
+const populationArr = [...populationList].map(el =>
+  +(el.innerText.split(',').join('')));
+const total = populationArr.reduce((a, b) => (a + b));
```

Comment:
it's better to set default value for reduce method and use some more understandable names
```suggestion
const total = populationArr.reduce((acc, number) => (acc + number), 0);
```